examples/subscription/gopay_subscription_example.py

```diff
@@ -0,0 +1,145 @@
+# This is just for very basic implementation reference, in production, you should validate the incoming requests and implement your backend more securely.
+import midtransclient
+import datetime
+
+# Initialize core api client object
+# You can find it in Merchant Portal -> Settings -> Access keys
+core_api = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+
+# To use API subscription for gopay, you should first link your customer gopay account with gopay tokenization
+# Refer to docs: https://api-docs.midtrans.com/#gopay-tokenization
+
+# You will receive gopay payment token with get_payment_account API
+# Check Tokenization examples folder (/examples/tokenization)
```

Comment:
why is this grammar not updated? When I told you to fix the description on the readme, intuitively you should also fix this, as the sentence is identical.